python/src/aiconfig/editor/client/src/components/prompt/RunPromptButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { ClientPrompt } from "../../shared/types";
+import { Button, createStyles, Flex, Text } from "@mantine/core";
+import { IconPlayerPlayFilled } from "@tabler/icons-react";
+import { memo } from "react";
+
+type Props = {
+  prompt: ClientPrompt;
```

Comment:
Is this used this diff?

Reply:
Not yet, but will be used in subsequent diff